packages/create-app/src/createApp.ts

```diff
@@ -145,7 +145,8 @@ export default async (cmd: Command): Promise<void> => {
 
     Task.log();
     Task.log(
-      chalk.green(`🥇  Successfully created ${chalk.cyan(answers.name)}`),
+      chalk.green(`🥇  Successfully created ${chalk.cyan(answers.name)}
+                   See https://backstage.io/docs/tutorials/quickstart-app-auth to know more about enabling auth providers`),
```

Comment:
Prolly go with multiple log lines instead of a single template string block. This second line is gonna be weirdly indented